daphne_worker/src/lib.rs

```diff
@@ -198,7 +203,7 @@ impl DaphneWorkerRouter {
         };
 
         let router = if self.enable_internal_test {
-            router.post_async("/internal/delete_all", |_req, ctx| async move {
+            router.post_async("/:version/internal/delete_all", |_req, ctx| async move {
```

Comment:
Internal APIs don't need to be versioned (and they shouldn't be).

Reply:
I've used a sort of a "hack-y" way of removing "version" from the URL.
I've added a note just in case someone is curious.